docs/_posts/2017-03-09-jekyll-3-4-2-released.markdown

```diff
@@ -0,0 +1,51 @@
+---
+title: 'Jekyll 3.4.2 Released'
+date: 2017-03-09 15:41:57 -0500
+author: parkr
+version: 3.4.2
+categories: [release]
+---
+
+Another one-PR patch update, though without the same [lessons as for the
+previous release]({% link _posts/2017-03-02-jekyll-3-4-1-released.markdown %}).
+
+This release includes a fairly critical change for a number of plugins:
```

Comment:
I wouldn't call it critical (e.g., having the potential to become disastrous). Perhaps important? beneficial?